packages/components/src/Alert/index.tsx

```diff
@@ -132,7 +140,7 @@ function Alert({
           <div className="nimbus--alert__details">
             <div className="nimbus--alert__body">
               {title && <Title type="h6">{title}</Title>}
-              <Text size="small">{children}</Text>
+              <div className="nimbus--alert__body_children">{Children}</div>
```

Comment:
Acá no tendría que ser `children` con la c minúscula? 👇🏼 

```suggestion
              <div className="nimbus--alert__body_children">{children}</div>
```

Reply:
Juanchi, ahí estoy tomando el componente de la línea 128, donde ahí se define qué código utilizar. Por eso va `Children`. 

Dentro de Children se encuentra la lógica que buscamos cumplir. 
Si el `children` es de tipo string, le agrega `Text` de lo contrario, deja directamente el `children` viene.

Reply:
Perfecto Nico, quedó clarísimo. Gracias 🙂